src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -71,13 +80,13 @@ const ChatItemMyTextV2 = ({
   const renderIcon = React.useCallback(() => {
     if (status === ChatStatus.PENDING)
       return (
-        <View style={styles.icon}>
+        <View style={styles.iconNewLine}>
           <IconChatClockGrey color={colors.silver} width={12} height={12} />
         </View>
       );
 
     return (
-      <View style={styles.icon}>
+      <View style={styles.iconNewLine}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the icon has been changed from `styles.icon` to `styles.iconNewLine`. Ensure that this change doesn't affect the layout in an undesired way.



<!-- This is an auto-generated comment by OSS CodeRabbit -->